### DISCUSSION TITLE
Minimums requirements to run KEDA

### DISCUSSION THREAD

**Discussion by Fillo22** [kedacore/keda#1658] [Q&A / Need Help] [answered]
Hi everyone,
I'm looking for minimum requirements in order to run Keda on a OpenShift Cluster. Does anyone have an idea on where can i find them?

Thanks

**Comment by tomkerkhove** [comment]
What requirements are you looking for?  We provide guidance in terms of cluster (https://keda.sh/docs/2.1/operate/cluster/) and firewal config is coming soon (https://github.com/kedacore/keda-docs/pull/390)

**Comment by zroubalik** [comment, marked as answer]
Here is the link: https://keda.sh/docs/2.1/operate/cluster/#cluster-capacity-requirements

KEDA itself doesn't have a big requirements, it mostly depends on the number of ScaledObjects.

**Reply by Fillo22** [reply]
Thank you for the fast and helpfull answer!
